frontend/components/icons/Lightbulb.tsx

```diff
@@ -0,0 +1,32 @@
+import React from "react";
+import { COLORS, Colors } from "styles/var/colors";
+
+interface ILightbulbProps {
+  color?: Colors;
+}
+
+const Lightbulb = ({ color = "core-fleet-blue" }: ILightbulbProps) => {
+  return (
+    <svg
+      width="28"
```

Comment:
Do we want to use ICON_SIZES here?

Reply:
@jacobshandling let's merge as is and if we want to modify the size we can iterate